modules/statics/src/coins.ts

```diff
@@ -286,7 +314,7 @@ export const coins = CoinMap.fromCoins([
     Networks.test.bitcoinGold,
     UnderlyingAsset.BTG,
     BaseUnit.BTC,
-    BTG_FEATURES
+    BTG_FEATURES.filter((f) => f !== CoinFeature.MULTISIG_COLD)
```

Comment:
technically we shouldn't even have this as an option in statics, since there is no such coin implementation for `tbtg`, only the mainnet one but I'd rather not open this change up to also include breaking changes if not necessary.